charter.md

```diff
@@ -2,92 +2,82 @@
+- Selective Disclosure of claims (supporting collection limitation and attribute redaction)
+- Zero Knowledge Proofs (demonstrating some properties about some attributes without disclosing their values)
+- Proof of ownership of a digital presentation (using a TA running in a TEE)
+- Detection or prevention of holders collaborative attacks
+- Authenticity (ensuring that the issuer of a digital credential can be either individually authenticated or authenticated as a member of a set of trusted issuers)
 - Integrity (ensuring tampering with information is always evident)
-- Confidentiality (ensuring information is protected from unauthorized access)
-- Hardware Assurance (awareness of the security capabilities of issuers, holders and verifiers)
-- Availability (efficiently information processing, minimizing data in flight, reducing carbon cost for storage and transmission, and supporting offline / paper credential use cases)
 - Semantic interchangeability (ensuring terms are understood and used consistently, in a global or local context)
-- Usability and Feasibility (the core operations must be easy to understand and use, and possible to integrate with existing platforms)
-- Accountability (what format and protocol considerations can help mitigate abusive or excessive requests for credential presentation?  What mitigates abuse by issuers?)
+- Usability (the core operations must be easy to understand and use)
 
 ## Goals
 
-Coordination with CFRG, RATS, OAuth, JOSE, COSE, and SCITT are important to ensure that the latest work at IETF is leveraged. Feedback from experts in other IETF working groups is gathered in the SPICE WG without creating fragments of credential work spread across several existing places in the IETF. Additionally, the SPICE WG will coordinate with other SDOs, such as TCG, CCC, ISO or W3C, on data model elements needed to support existing credential use cases.
+Coordination with CFRG, RATS, OAuth, JOSE, COSE, and SCITT are important to ensure that the latest work at IETF is leveraged.
+Feedback from experts in other IETF working groups is gathered in the SPICE WG without creating fragments of credential work s
+pread across several existing places in the IETF. Additionally, the SPICE WG will coordinate with other SDOs, such as ISO or W3C,
+on data model elements or protocols needed to support existing credential use cases.
 
-### In-Scope
+## In-Scope
 
-The work of the SPICE WG aligns JWT and CWT registered claim names for use with the 'Three Role Model' as introduced by [W3C's 'Verifiable Credentials Data Model v2.0' specification](https://w3c.github.io/vc-data-model/#roles).
+The SPICE WG will develop a framework in support of the 'Three Role Model', considering in particular selective disclosure of attributes
+and unlinkability between verifiers. It will consider the use of TEEs (Trusted Execution Environments) for managing key material and digital credentials.
```

Comment:
```suggestion
The SPICE WG will develop a framework in support of the 'Three Role Model', considering in particular selective disclosure of attributes and unlinkability between verifiers. It will consider the use of TEEs (Trusted Execution Environments) for managing key material and digital credentials.
```

Reply:
Just formatting